materials/r-sql.md

```diff
@@ -5,11 +5,17 @@ title: Integrating R and SQL
 language: R
 ---
 
-* Already seen how to connect to database using `dplyr`
-* What if we want to do this outside of a `dplyr` context
-* Use `DBI` and `RSQLite` (or other appropriate DBMS plugin)
+> Remember to
+>    
+> * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171). *Make sure it does not have `SpeciesCounts` view.*
```

Comment:
Oh yeah. I can see that happening after a practice run. How about "*Make sure the copy you are going to use in class does not have..."